web/template/partial/shortcuts.gohtml

```diff
@@ -0,0 +1,87 @@
+                <div class="flex bg-gray-100 dark:bg-secondary-light text-5 px-2 py-1 rounded mr-3">
+                    <span class="text-xs font-semibold my-auto">j</span>
+                </div>
+                <p class="text-sm align-middle m-0">Seek backwards (10 seconds)</p>
+            </div>
+            <div class="flex justify-between">
+                <div class="flex bg-gray-100 dark:bg-secondary-light text-5 px-2 py-1 rounded mr-3">
+                    <span class="text-xs font-semibold my-auto">l</span>
+                </div>
+                <p class="text-sm align-middle m-0">Seek forwards (10 seconds)</p>
+            </div>
+            <div class="flex justify-between">
+                <div class="flex bg-gray-100 dark:bg-secondary-light text-5 px-2 py-1 rounded mr-3">
+                    <span class="text-xs font-semibold my-auto">f</span>
+                </div>
+                <p class="text-sm align-middle m-0">Open fullscreen</p>
+            </div>
+            <div class="flex justify-between">
+                <div class="flex bg-gray-100 dark:bg-secondary-light text-5 px-2 py-1 rounded mr-3">
+                    <i class="text-xs fa-solid fa-arrow-up my-auto"></i>
+                </div>
+                <p class="text-sm align-middle m-0">Increase volume</p>
+            </div>
+            <div class="flex justify-between">
+                <div class="flex bg-gray-100 dark:bg-secondary-light text-5 px-2 py-1 rounded mr-3">
+                    <i class="text-xs fa-solid fa-arrow-down my-auto"></i>
+                </div>
+                <p class="text-sm align-middle m-0">Decrease volume</p>
+            </div>
+            <div class="flex justify-between">
+                <div class = "flex">
+                    <div class="flex bg-gray-100 dark:bg-secondary-light text-5 px-2 py-1 rounded">
+                        <span class="text-xs font-semibold my-auto">1</span>
+                    </div>
+                    <span class = "font-semibold my-auto mx-2">-</span>
+                    <div class="flex bg-gray-100 dark:bg-secondary-light text-5 px-2 py-1 rounded mr-3">
+                        <span class="text-xs font-semibold my-auto">9</span>
+                    </div>
+                </div>
+                <p class="text-sm align-middle m-0">Seek to the 10% - 90% of the video</p>
+            </div>
+            <div class="flex justify-between">
+                <div class="flex bg-gray-100 dark:bg-secondary-light text-5 px-2 py-1 rounded">
+                    <span class="text-xs font-semibold my-auto">0</span>
+                </div>
+                <p class="text-sm align-middle m-0">Seek to the beginning of the video</p>
+            </div>
+            <div class="flex justify-between">
+                <div class="flex bg-gray-100 dark:bg-secondary-light text-5 px-2 py-1 rounded">
+                    <span class="text-xs font-semibold my-auto"> < </span>
```

Comment:
Would this be safer or does this make no difference?
```suggestion
                    <span class="text-xs font-semibold my-auto"> &lt; </span>
```

Reply:
Yup. That's definitely an improvement and less confusing.